src/main/java/com/example/MG_RestaurantManager20/product/domain/Product.java

```diff
@@ -15,8 +24,7 @@ public class Product {
     private Double quantity;
     private ProductUnit productUnit;
 
-    public Product() {
-    }
+    //private Recipe recipeDummy;
```

Comment:
Delete this comment if we don't need this

Reply:
Done kierowniku